client/src/header.css

```diff
@@ -0,0 +1,52 @@
+#head{
+    position: fixed;
+    display: inline;
+    margin-top: 25px;
+    margin-left: 0px;
+    width: 100vw;
+    height: 104px;
+    opacity: 1;
+}
+
+#camera{
+    align-content: right;
+    margin-top: 40px;
+    margin-bottom: 33px;
+    margin-right: 10vmin;
+    margin-left: 86vmin;
+    width: 48px;
+    height: 48px;
+    background: transparent url('./camera.png') 0% 0% no-repeat padding-box;
+    opacity: 1;
+}
+
+#insta{
+    padding-top: 0px;
+    justify-content: center;
+    display: inline;
+    margin-top: 25px;
+    margin-left: 14.2vmin;
```

Comment:
what is vmin ?

Reply:
vmin is minimum of height/width

Reply:
but why are you using vmin here, does not seems to make sense here

Reply:
so that the GUI becomes independent of frame sizes